heft-plugins/heft-sass-plugin/src/SassPlugin.ts

```diff
@@ -26,6 +26,7 @@ export interface ISassConfigurationJson {
   fileExtensions?: string[];
   nonModuleFileExtensions?: string[];
   silenceDeprecations?: string[];
+  excludeFiles?: string[];
```

Comment:
Should this be a glob pattern? I'm not attached to that idea, but we have done similar in the past for other plugins.

Reply:
These are globs; this already existed on the schema it just wasn't being forwarded.